obm-autoset/applyobmsetting.py

```diff
@@ -0,0 +1,350 @@
+'''Copyright 2016, DELL|EMC, Inc.
+
+Author(s):
+Norton Luo
+
+
+RackHD OBM setting automatic set utility
+'''
+
+
+import os
+import sys
+
+# External imports
+try:
+    import requests
+except IOError:
+    print "Python 'requests' package required for this script." \
+          "\nUse: 'pip install requests'\nExiting...\n"
+    exit(1)
+try:
+    import pexpect
+except IOError:
+    print "Python 'pexpect' package required for this script." \
+          "\nUse: 'pip install pexpect'\nExiting...\n"
+    exit(1)
+
+# Check if CI_TOOLS_PATH environment variable is set.  If not, take a stab that the
+# test-tools stash repository is located at same level as tests repository in the user environment
+# _no_tools_repository = False
+# _path_add = os.environ.get('CI_TOOLS_PATH',"../../test-tools")
+# sys.path.append(_path_add)
+# try:
+#     import reporter
+#     import reporter.utils
+# except ImportError:
+#     _no_tools_repository = True
+
+
+# Standard imports
+import json
+import argparse
+import subprocess
+import time, datetime
+import sys
+import unittest
+import signal
+import argparse
+#Load OBM config file
+try:
+    OBM_CONFIG = json.loads(open("obm_config.json").read())
+except:
+    print "**** Global Config file: " + "global_config.json" + " missing or corrupted! Exiting...."
+    sys.exit(255)
+
+
+def get_auth_token():
+    # This is run once to get an auth token which is set to global AUTH_TOKEN and used for rest of session
+    global AUTH_TOKEN
+    global REDFISH_TOKEN

```

Comment:
I think your script will not use the redfish API, so please remove related code.